readme/rich_text_editor.md

```diff
@@ -14,6 +14,8 @@ However **there is a catch**: in Joplin, notes, even when edited with this Rich
 
 - Tables must have a header, because this is a requirement in Markdown. When you create a table, it will let you create it without a header, but under the hood it will add an empty one. And next time you open the note, this empty header will be present.
 
+- List items (bullet points, numbered lists & checkboxes) within table cells can be created but will not be saved correctly; lists within tables are not currently part of any Markdown specifications supported by Joplin.
```

Comment:
`not currently part of any Markdown specifications supported by Joplin.`

Is there any markdown spec that allows that? If no, we should change it to:

```suggestion
- List items (bullet points, numbered lists & checkboxes) within table cells can be created but will not be saved correctly; lists within tables are not currently part of any Markdown specification.
```

Reply:
Neither am I to be honest but seeing how common it is I'm surprised if there isn't an unknown flavour of it out there already or if there is an flavour being written that includes it. Didn't want to be too specific only to be corrected two weeks later by a link to an overly ambitious markdown flavour project somewhere.
Happy to change it to your suggestion otherwise.

Reply:
I rather think it's because people don't know the spec and maybe some renderers have a bug that allows this.
Wording it the way you did will lead to the following: please support the spec that allows it.

There is only one markdown spec. The flavors are not really our concern. Users can be happy that we support a few of the GH flavor choices.
If we add parts of X different flavors, the file would not be markdown anymore.

I believe this is also the reason why we don't do a Joplin flavor.

Reply:
And people don't seen to understand that the WYSIWYG editor still produces a markdown file.

Reply:
> Wording it the way you did will lead to the following: please support the spec that allows it.
Agree totally, didn't think of it that way, I think thats the main thing here

The rest I also agree, its also why I'm fairly strict on my plugin usage that requires custom tags, it deviates from the standard so now my notes are a weird kind of propriatary and not true Markdown.

Reply:
We could even have written "not part of the CommonMark specification". In general that's what we follow (as much as we can) and if it was supported, markdown-it would support it too, and so would we.

Reply:
We can change it if you prefer.